Assets/Scripts/StartupController.cs

```diff
@@ -18,7 +18,7 @@ public class StartupController : MonoBehaviour {
 
 	// Runs on app startup - start server connection, login, load game data
 	void Start () {
-		//PlayerPrefs.DeleteKey("session"); // Uncomment this to test from login screen
+		PlayerPrefs.DeleteKey("session"); // Uncomment this to test from login screen
```

Comment:
I uncomment this so that we **can** login as a different user - it makes you login every time, which is annoying but rather required for the level of testing we're doing